event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepository.java

```diff
@@ -226,19 +223,16 @@ public boolean recordExists(final long offset, final UUID streamId) {
         }
     }
 
-    public Stream<Event> first(final long pageSize,
-                               final UUID streamId)
+    public Stream<Event> first(final long pageSize, final UUID streamId)
```

Comment:
again, streamID is the key field here... how many we return is less important.... so order params